src/Segments.hs

```diff
@@ -39,7 +39,8 @@ segmentHandlers = Map.fromList [
         ("powerline.segments.shell.jobnum",                 Segments.Shell.jobNumSegment),
         ("powerline.segments.shell.last_pipe_status",       Segments.Shell.pipeStatusSegment),
         ("powerline.segments.shell.last_status",            Segments.Shell.lastStatusSegment),
-        ("powerline.segments.shell.mode",                   Segments.Shell.modeSegment)
+        ("powerline.segments.shell.mode",                   Segments.Shell.modeSegment),
+        ("powerline_gitstatus.gitstatus",                   Segments.VCS.gitCommitsAheadCountSegment)
```

Comment:
It would be worthwhile adding a comment here to explain why this has a different name from all the other VCS segments.